AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/BundleHelper.java

```diff
@@ -172,6 +173,10 @@ public BundleHelper(AndroidAnnotationsEnvironment environment, TypeMirror elemen
 			if (isTypeParcelable(type)) {
 				methodNameToSave = "put" + "Parcelable";
 				methodNameToRestore = "get" + "Parcelable";
+			} else if (annotationHelper.isAnnotatedWith(type, "org.parceler.Parcel")) {
```

Comment:
@johncarl81 it is very easy to check it is on the classpath: `elementUtils.getTypeElement("org.parceler.Parcer") != null`. We use this type of check for lots of things throughout AA.